packages/gatsby-core-utils/src/get-term-program.ts

```diff
@@ -0,0 +1,15 @@
+import { isCI } from "."
+
+export function getTermProgram(): string | undefined {
+  if (isCI()) {
```

Comment:
maybe we could always capture it?

Reply:
I'm not sure what value or meaning it would have, but if you're happy to filter them out of reports then in happy for us to send it 😁